packages/fast-components-react-msft/src/slider-label/README.md

```diff
@@ -1,5 +1,23 @@
-# Slider label
+## Slider label
+*Slider labesls* are styled label versions of base *Slider track item" components positioned at specific values on a slider's track to act as labels.  By default a *slider label* renders above the *slider*'s front and back bars and below the thumbs.
 
-## Usage
+### Usage
+*Slider label* exposes a "position binding" prop which can either be a numerical value that falls within the parent *slider*'s range or a SliderTrackItemAnchor enum value.  
 
-Displays a tick mark and optional label for a *slider* value.
+When a *slider label* renders as the child of a *slider* it uses absolute positioning to place itself at a position that corresponds to the value specified by the maxValuePositionBinding prop on the slider's track. 
```

Comment:
Can you add backticks around prop names and enums to make them stand out from the rest of the text so this is a little more readable?

Reply:
yep